tests/test_match.py

```diff
@@ -1887,25 +1939,29 @@ def test_match_all_forecasts_1(this, self):
         self.assertEqual(sphinx.sep_match_status[self.threshold_key], 'SEP Event', '')
         self.assertEqual(sphinx.end_time_match_status[self.threshold_key], 'SEP Event', '')
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-    
-    
-    
-    
-    
+    @make_docstring_printable
+    def test_match_all_forecasts_2(this, self):
+        """
+        The forecast/observation pair has the following attributes:
+           -- Prediction flux threshold is > 3 pfu
+           -- Prediction energy threshold is > 10 MeV
+           -- Observation flux threshold is > 10 pfu
+           -- Observation energy threshold is > 10 MeV
+           -- Prediction window overlaps with observation window
+                prediction window start:  2000-01-01T00:00:00Z
+                prediction window end:    2000-01-01T01:00:00Z
+                observation window start: 2000-01-01T00:00:00Z
+                observation window end:   2000-01-01T01:00:00Z
+           -- No ongoing SEP event at start of prediction window
+                event start:              2000-01-01T00:15:00Z
+                event end:                2000-01-01T00:20:00Z
+           -- The trigger (CME observation) for the forecast occurred before the threshold crossing 
+                CME start:                2000-01-01T00:00:00Z
+           -- A threshold crossing occurred within the prediction window
+                threshold crossing:       2000-01-01T00:15:00Z
+        sphinx.all_clear_match_status should be 'No Matching Threshold'
+        """
+        forecast_jsons = ['./tests/files/forecasts/match/match_all_forecasts/match_all_forecasts_2.json']
+        matched_sphinx, all_obs_thresholds, observed_sep_events = self.utility_test_match_all_forecasts(this, forecast_jsons)
+        sphinx = matched_sphinx['unit_test'][self.energy_key][0]
+        self.assertEqual(sphinx.all_clear_match_status, 'No Matching Threshold', '')
```

Comment:
I was confused about the null string `''` as the third argument.  Looks like this is setting `msg=''`, removing a specific error message.  Why is this better than leaving it as the default `None`?  If it is better, suggest specifying the keyword explicitly instead of by position, e.g. write `msg=''`.

Reply:
I think the inclusion of the null string `''` as the third argument arose because I was initially under the impression that `msg` was a required argument (because I'd lifted the `assertEqual()` calls from an example). I can remove the null strings from all `assertEqual()` calls if you want (potentially in a separate pull request)?

Reply:
Change it to omit it & use the default here, but change it for the other unit tests in another dedicated PR.